.credo.exs

```diff
@@ -81,7 +81,7 @@
         {Credo.Check.Readability.ModuleAttributeNames},
         {Credo.Check.Readability.ModuleDoc},
         {Credo.Check.Readability.ModuleNames},
-        {Credo.Check.Readability.ParenthesesOnZeroArityDefs},
+        {Credo.Check.Readability.ParenthesesOnZeroArityDefs, false},
```

Comment:
I think `mix format` enforces this and we should start using mix formatter in this project at some point in time but probably after this is merged in :)

Reply:
I've been formatting the files as I touch them. 

There isn't a super good way to do this but i dislike having a "format the whole project" commit because it makes git blame useless sometimes.

Reply:
I mean not useless you can still git blame past that commit.

I think having a whole "mix format this" commit is the best, while not ideal. It allows you to enable the check in CI and the longer you work without it the more files will have that git blame problem.

Also, formatting while touching the files has the same problem (- the functions that were actually changed) because then a change is there but it's also not related to the commit. It's just more dispersed and harder to spot.

Reply:
Probably just need to bite the bullet at some point.

Reply:
Contrarily, we could add a .formatter.exs file with no input files specified which should disable the formatter assuming that people's editors aren't doing something shitty and ignoring the .formatter file.